crosswalks/ogcapi-coverages.md

```diff
@@ -0,0 +1,428 @@
+are condensed into the same **cube:dimensions** container.
+
+### STAC
+
+**`catalog`**
+A logical group of other **catalog**, **collection**, and **item** objects.
+
+**`collection`**
+A set of common fields to describe a group of **items** that share properties and metadata
+
+**`item`**
+A GeoJSON Feature augmented with foreign members relevant to a STAC object.
+
+**`cube:dimensions`**
+A set of dimensions that can be of different types: spatial, temporal, "other", etc.
+
+
+## Crosswalk
+
+*Executive summary (tl;dr)*
+
+The capabilities of **openEO** and **OAC1** largely overlap,
+though, as also declared in the standard draft's preface, **OAC1** is a simpler
+service, with limited functionality in its core specification
+(eg. no authentication/authorization management, processes,
+batch jobs, logs, processing budgets, etc).
+
+**OAC1** usually provide simpler ways to request basic operations on 
+datasets (like filtering or resampling), by using the query parameters of
+the URLs. **openEO** on the other hand allows for a much powerful service
+with processing graphs and asynchronous jobs management, at the cost of a
+slightly more laborious way to execute the most basic "processing" operations.
+
+The data model of the "coverage" is generally a bit more flexible with regards
+to the "arbitrary" non-spatial non-temporal dimensions, while the processes
+in **openEO** generally do not include such dimensions.
+
+The only **OAC1** capability that **openEO** does not directly provide (if not
+via cascade with a secondary tiles service) is to attach tilesets to coverages, and hence
+enable tiled access to the data.
+
+
+<a name="meta-api-capabilities"/>
+
+### Meta / API capabilities
+
+In this section we go through the available information on the API capabilities,
+very relevant for interoperability and "machine-actionable" services.
+
+### Well-known URIs
+
+- openEO: [`GET /.well-known/openeo`](https://api.openeo.org/1.2.0/#tag/Capabilities/operation/connect) (optional)
+
+- OAC1: `n/a`
+
+Site-wide metadata information, in **openEO** this is a list of versions of the API that the server supports,
+and it is meant to help a client choose the most suitable endpoint (eg. production/development).
+
+Not available in **OAC1**.
+
+
+### Landing page
+
+- openEO: [`GET /`](https://api.openeo.org/1.2.0/#tag/Capabilities/operation/capabilities) (required)
+
+- OAC1: `GET /` (required)
+
+Both landing pages stem from the [OGC API - Common](https://ogcapi.ogc.org/common/) standard,
+and they only have minor differences in the returned schemas.
+
+### Conformance
+
+- openEO: [`GET /conformance`](https://api.openeo.org/1.2.0/#tag/Capabilities/operation/conformance) (optional) (only from v1.2.0)
+
+- OAC1: `GET /conformance` (required)
+
+Both endpoints are 100% equivalent (**openEO** additionally allows to list the conformance classes in the landing page,
+following the STAC API).
+
+
+### API description
+
+- openEO: `n/a` (?)
```

Comment:
The way of linking to OpenAPI documents in openEO and OAC1 is the same, it's just that OAC1 pre-defines a path for it and requires the links, which are optional in openEO.

Reply:
So in openEO there is no predefined link to the OpenAPI docs? How can a client discover the link?
(feel free to edit the document yourself anyway, of course)

Reply:
In both specs, you use the relation types `service-desc` and `service-doc`. I think in both cases they can point anywhere, but OGC APIs propose to use `/api` as endpoint.

Reply:
(It's better to first discuss and come to a common understanding and update afterwards :-) )